### PR TITLE
Feature/sc 5683 move rest ofthe homework.js to the courses po and adapt tests

### PR DESCRIPTION
**Pull request by TomaszJuchimiuk**
# Description
-removed homeworks.js,
-moved all methods from homeworks.js to new structure,

## Approval for review
- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: page-objects/pages/coursePages/CRSSCourseListPage.js [hunk 1]
```diff
@@ -3,6 +3,9 @@
 const { CLIENT } = require("../../../shared-objects/servers")
 const eh = require("../../../runtime/helpers/elementHelpers");
 const wh = require("../../../runtime/helpers/waitHelpers");
+const courseData = require('../../../shared-objects/courseData');
+const firstLogin = require('../../../shared_steps/firstLogin.js');
+const elementHelpers = require('../../../runtime/helpers/elementHelpers.js');
```
Comment (yuliashikhareva) [inline page-objects/pages/coursePages/CRSSCourseListPage.js:8]: duplicated import (elementHelpers, eh)

File: page-objects/pages/coursePages/CRSSCourseListPage.js [hunk 2]
```diff
@@ -3,6 +3,9 @@
...
 const { expect } = require("chai");
```
Comment (yuliashikhareva) [inline page-objects/pages/coursePages/CRSSCourseListPage.js:9]: expect doesn't need to be imported as globally available

File: page-objects/pages/coursePages/CRSSCourseListPage.js [hunk 3]
```diff
@@ -11,6 +14,7 @@ const searchCourseFiled = ".input-group .search-field";
 const courseWrapper = ".sc-card-wrapper";
 const titleOfCourse = ".title";
 const memberBtn = ".btn-member";
+const homeworktab = '.tabs button[data-testid="hometasks"]';
```
Comment (yuliashikhareva) [inline page-objects/pages/coursePages/CRSSCourseListPage.js:17]: I think that it is always better to keep selectors together as one object, like const selectors= {selector1, selector2...}. and the add selectors to module.exports. What is your opinion? as I did refactoring I tried to keep it everywhere  like that.

File: page-objects/pages/coursePages/CRSSCourseListPage.js (hunk 1 above)
Comment (TomaszJuchimiuk) [reply page-objects/pages/coursePages/CRSSCourseListPage.js:8]: done

File: page-objects/pages/coursePages/CRSSCourseListPage.js (hunk 2 above)
Comment (TomaszJuchimiuk) [reply page-objects/pages/coursePages/CRSSCourseListPage.js:9]: done

File: page-objects/pages/coursePages/CRSSCourseListPage.js (hunk 3 above)
Comment (TomaszJuchimiuk) [reply page-objects/pages/coursePages/CRSSCourseListPage.js:17]: done

**Review by Grzesiek80 — APPROVED** [review]

**Pull request merged**